plugins/janus_streaming.c

```diff
@@ -1331,6 +1331,7 @@ json_t *janus_streaming_query_session(janus_plugin_session *handle) {
 	if(mp) {
 		json_object_set_new(info, "mountpoint_id", json_integer(mp->id));
 		json_object_set_new(info, "mountpoint_name", mp->name ? json_string(mp->name) : NULL);
+		json_object_set_new(info, "mountpoint_listeners", mp->listeners ? json_integer(g_list_length(mp->listeners)) : NULL);
```

Comment:
This should be protected by `mp->mutex`, and I'd put a `json_integer(0)` in there instead of NULL if there are no listeners currently.

Reply:
Thanks, updated!